configurations/performance/c4/c4-highmem-16-lssd.yaml

```diff
@@ -0,0 +1,9 @@
+gce_instance_type_db: 'c4-highmem-16-lssd'
+gce_root_disk_type_db: 'hyperdisk-balanced' # those doesn't persistent disks
+gce_n_local_ssd_disk_db: 2 # fpr some reason it forces to specify this value (should try 0)
```

Comment:
strangely, I didn't have to specify it (I used 0)

Reply:
it works with both

when I use it with the default the error message said it should be 2, and with 2 it also worked